src/Symfony/Bundle/DebugBundle/composer.json

```diff
@@ -23,9 +23,12 @@
     },
     "require-dev": {
         "symfony/config": "~2.8|~3.0",
-        "symfony/dependency-injection": "~2.8|~3.0",
+        "symfony/dependency-injection": "~3.3",
```

Comment:
hmm indeed, they get it to get the template for the profiler, which is not in WebProfilerBundle unlike other collectors